### PR TITLE
remove the property reference on `ErrorDetail` and `ErrorAdditionalInfo`

### PR DESCRIPTION
**Pull request by ArcturusZhang**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/resourcemanager/Azure.ResourceManager/src/autorest.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ directive:
   - remove-model: "CheckNameAvailabilityRequest"
   - remove-model: "CheckNameAvailabilityResponse"
   - remove-model: "ErrorResponse"
+  - remove-model: "ErrorDetail"
+  - remove-model: "ErrorAdditionalInfo"
```
Comment (ArthurMa1978) [inline sdk/resourcemanager/Azure.ResourceManager/src/autorest.md:67]: update the apiview

**Review by Yao725 — APPROVED** [review]

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**